recipes/recipes_schubar.json

```diff
@@ -841,6 +841,30 @@
         "ice": "cubed",
         "prep": "build",
         "garnish": "Flag of lemon peel and raspberries"
+    },
+
+    "Cayuga White": {
+        "info": "Vintage 2013",
+        "tag": "core",
+        "unit": "oz",
+        "prep": "cool as desired",
```

Comment:
"prep" is kind of a special attribute that has to come from a restricted list, so just use "pour"